ansible/roles/webserver/tasks/main.yml

```diff
@@ -1,18 +1,23 @@
 ---
-- name: php repository gpg key (stretch only)
+- name: php repository gpg key (buster only)
   become: yes
   apt_key:
     url: "https://packages.sury.org/php/apt.gpg"
     state: present
-  when: raspbian_version.stdout == 'stretch'
+  when: raspbian_version.stdout == 'buster'
 
-- name: php repository (stretch only)
+- name: php repository (buster only)
   become: yes
   apt_repository:
-    repo: deb https://packages.sury.org/php/ stretch main
+    repo: deb https://packages.sury.org/php/ buster main
     filename: php7.list
     state: present
-  when: raspbian_version.stdout == 'stretch'
+  when: raspbian_version.stdout == 'buster'
+
+- name: Upgrade numpy with pip
+  pip: 
+    name: numpy
+    extra_args: --upgrade
```

Comment:
I am pulling this discussion here rather than on #503 as I think it's a better place for it.

@dom-robinson Why do we need to this horrible hack?!

Reply:
Wait, I think [this](https://github.com/jekhokie/raspberry-noaa-v2/blob/master/requirements.txt#L6) is the place that needs updating. What version of numpy do we need?

Reply:
I seriously disapprove of this hybrid system / pip python setup we have going on. It's a recipe for chaos.